### DISCUSSION TITLE
[新手求助]请问使用该项目时如何获取conversation_id

### DISCUSSION THREAD

**Discussion by Wu-jiyan** [linweiyuan/go-chatgpt-api#213] [Q&A] [answered]
我想要知道如何使用这个项目来获取chatgpt官方的conversation_id来进行对话
如果可行的话请大佬告诉我，谢谢！

**Comment by linweiyuan** [comment, marked as answer]
https://github.com/linweiyuan/go-chatgpt-api/issues/121

**Reply by Wu-jiyan** [reply]
啊谢谢谢谢谢谢大佬
刚没看到示例，抱歉麻烦了
